libsolidity/formal/SymbolicTypes.cpp

```diff
@@ -139,7 +140,22 @@ pair<bool, shared_ptr<SymbolicVariable>> newSymbolicVariable(
 	else if (isBool(_type.category()))
 		var = make_shared<SymbolicBoolVariable>(type, _uniqueName, _context);
 	else if (isFunction(_type.category()))
-		var = make_shared<SymbolicFunctionVariable>(type, _uniqueName, _context);
+	{
+		auto const& fType = dynamic_cast<FunctionType const*>(type);
+		auto params = fType->parameterTypes() + fType->returnParameterTypes();
```

Comment:
Instead of concatenating the two vectors here, you could move everything below into a
``auto doTheStuff = [&](auto&& params) { ... }`` and then call ``doTheStuff(fType->parameterTypes());`` followed by ``doTheStuff(fType->returnParameterTypes());`` - that should fix MSVC being weird and it also avoids copying, so it's actually more efficient anyways (and I think it's not much harder to read, is it?).